Documentation/devel/release.md

```diff
@@ -1,9 +1,22 @@
 # rkt release guide
 
-How to perform a release of rkt.
-This guide is probably unnecessarily verbose, so improvements welcomed.
-Only parts of the procedure are automated; this is somewhat intentional (manual steps for sanity checking) but it can probably be further scripted, please help.
+## Release cycle
+
+This section describes the typical release cycle of rkt:
+
+1. A [milestone](https://github.com/coreos/rkt/milestones) is created on GitHub, fixing the target date for a future release (typically every 2/3 weeks).
```

Comment:
1. A GitHub [milestone](https://github.com/coreos/rkt/milestones) sets the target date for a future rkt release. Releases occur approximately every two to three weeks.